### PR TITLE
Ui/missing themeing tabs (#3)

### PR DESCRIPTION
**Pull request by vyvil**
**Can be reviewed now.**

This PR adds theme to Tabs, ActivityFeed and Button components.

### CODE REVIEW

File: src/components/ActivityFeed/ActivityFeed.js [hunk 1]
```diff
@@ -623,24 +625,26 @@ class ActivityFeed extends React.Component<Props, State> {
       maxScrollOffset,
     } = this.state;
 
+    const firstTab = tabs.length ? tabs[0].id : '';
```
Comment (poocart) [inline src/components/ActivityFeed/ActivityFeed.js]: as `tabs` is optional prop, maybe `lodash.isempty`?

File: src/components/Button/Button.js [hunk 2]
```diff
@@ -338,32 +372,38 @@ class Button extends React.Component<Props, State> {
...
     const {
       disabled,
       disabledTransparent,
       children,
       isLoading,
       style = {},
+      theme,
     } = this.props;
 
+    const updatedColors = themeColors(theme)[(getThemeType(this.props, true))];
```
Comment (poocart) [inline src/components/Button/Button.js]:
```suggestion
    const updatedColors = themeColors(theme)[getThemeType(this.props, true)];
```

File: src/components/Button/Button.js [hunk 3]
```diff
@@ -338,32 +372,38 @@ class Button extends React.Component<Props, State> {
...
-      <ButtonText customTheme={customTheme} small={small} style={textStyle}>
+      <ButtonText theme={updatedTheme} customTheme={customTheme} small={small} style={textStyle}>
         {title}
       </ButtonText>
     );
   };
 
   render() {
-    const customTheme = getTheme(this.props);
+    // const customTheme = getTheme(this.props);
```
Comment (poocart) [inline src/components/Button/Button.js]:
```suggestion
```

Comment (vyvil) [reply src/components/Button/Button.js]: I swear I've deleted this line :)) Will remove it. Thanks!

**Pull request merged**